src/app/game/game.component.html

```diff
@@ -11,7 +11,10 @@
           <div *ngFor="let square of shipMap">
             <mat-grid-tile>
               <button mat-mini-fab color="" 
-                [ngClass]="{'button-miss': square.status==2, 'button-hit': square.status==1, 'button-ship': square.status==3}">
+                [ngClass]="{'button-miss': square.status==2,
```

Comment:
Change needed to reflect introduced enum

Reply:
separate task created for that issue